data/mupen64plus.ini

```diff
@@ -13107,6 +13107,14 @@ GoodName=Star Wars - Rogue Squadron (U) (M3) [t1]
 CRC=165CBDA2 05A62246
 RefMD5=47CAC4E2A6309458342F21A9018FFBF0
 
+[2E458D7CC355D7918493B0E0362C9A20]
+GoodName=Star Wars - Rogue Squadron (U) (M3) (V1.1) [!]
+CRC=C7F30CFA ECB0FA36
+Players=1
```

Comment:
Just adding the GoodName, CRC and
RefMD5=47CAC4E2A6309458342F21A9018FFBF0
should also work and avoid duplication.